frontend/model/state.js

```diff
@@ -49,7 +50,8 @@ const state = {
   currentGroupId: null,
   contracts: {}, // contractIDs => { type:string, HEAD:string } (for contracts we've successfully subscribed to)
   pending: [], // contractIDs we've just published but haven't received back yet
-  loggedIn: false // false | { name: string, identityContractId: string }
+  loggedIn: false, // false | { name: string, identityContractId: string }
+  theme: 'dark'
```

Comment:
The default theme should be blue as per @mmbotelho's [design system](https://www.figma.com/file/ZR0DRPhhUuHdJtf7PbGAM6tW/Group-Income?node-id=1%3A7).

Reply:
this is fixed now